radixdlt/test.properties

```diff
@@ -0,0 +1,5 @@
+#
```

Comment:
Oops.  This file shouldn't be here.

Reply:
Ahh, this cunning beast finally discovered how to slip into commit :) Removed it